firebase_admin/_auth_client.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, app, tenant_id=None):
         if emulator_host:
             base_url = 'http://{0}/identitytoolkit.googleapis.com'.format(emulator_host)
             endpoint_urls['v1'] = base_url + '/v1'
-            endpoint_urls['v2beta1'] = base_url + '/v2beta1'
+            endpoint_urls['v2'] = base_url + '/v2'
```

Comment:
Did we test this change with on the emulator?

Reply:
Yes